docker/php/Dockerfile

```diff
@@ -75,12 +75,10 @@ RUN apt-get install -y apt-transport-https \
     && apt-get install yarn
 
 # Install XDEBUG
-RUN pecl install xdebug
-#    && docker-php-ext-enable xdebug \
-#    && echo "xdebug.remote_enable=1" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
-#    && echo "xdebug.remote_connect_back=1" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
-#    && echo "xdebug.idekey=\"PHPSTORM\"" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini \
-#    && echo "xdebug.remote_port=9001" >> /usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini
+RUN yes | pecl install xdebug \
```

Comment:
It would be simpler to just use the following lines :  

RUN pecl install xdebug \
    && docker-php-ext-enable xdebug

The next lines would be better of in the php.ini file